vertica_python/vertica/messages/backend_messages/notice_response.py

```diff
@@ -63,22 +63,26 @@ class NoticeResponse(BackendMessage):
     def __init__(self, data):
         BackendMessage.__init__(self)
         self.values = {}
+        self.attributes = {}
 
         pos = 0
         while pos < len(data) - 1:
             null_byte = data.find(b'\x00', pos)
 
             unpacked = unpack_from('c{0}sx'.format(null_byte - 1 - pos), data, pos)
             key = unpacked[0]
-            value = unpacked[1]
+            raw_value = unpacked[1]
+            value = raw_value.decode('utf-8') if type(raw_value) == bytes else raw_value
 
             self.values[FIELD_NAMES[key]] = value
             pos += (len(value) + 2)
 
         # May want to break out into a function at some point
```

Comment:
Would this be a useful diff to add the well-known fields as property getters to a base class?